src/components/ResultViewer.js

```diff
@@ -23,7 +24,12 @@ export class ResultViewer extends React.Component {
   static propTypes = {
     value: PropTypes.string,
     editorTheme: PropTypes.string,
+    ResultsTooltip: PropTypes.any,
   };
+  constructor() {
+    super();
+    this._tooltip = document.createElement('div');
```

Comment:
`codemirror-graphql` expects to recieve actual DOM element (not virtual dom) and I did not want to create new one every time tooltip is rendered (which happens every time mouse moves from character to different one). This has benefit that if tooltip remains unchanged when it moves it should not touch DOM (apart from moving it). We would loose that if we created new div every time.